src/modules/navBar.js

```diff
@@ -0,0 +1,27 @@
+import logo from '../../asset/resource/logo.jpeg';
+
+const renderNavBar = () => {
+  const nav = document.querySelector('.nav-container');
+  const navs = `<ul class="nav">
+        <li class='list-logo'>
+          <a href="#">
+            <img src=${logo} class="logo" alt="Event logo" />
+          </a>
+          <span class="ename">Evenue</span>
+        </li>
+        <li>
+          <a href="#">Events</a>
+        </li>
```

Comment:
- [ ] There should be a counter here to show the total number of events fetched from the API as stated in the requirement `We have counters in all the interfaces that show: The number of items (home).` Kindly add it.